Dashboard rev2

19etweinstock opened the pull request:
## Part Description
Devices:
- YC248-JR-0710KL: new 10K 8-resistor array to replace Bourns CAT16 Series 8x which is at end of life cycle

## Additional Information
Datasheets
-[YAG_YC248-M](https://www.yageo.com/upload/media/product/productsearch/datasheet/rchip/PYu-YC_TC_group_51_RoHS_L_9.pdf)

## Checklist
- [x] Did you create any new schematics or boards?
- - [ ] Did you *make a PR* for them in `circuits-2020`? If so, please pause until you get this PR merged.
- [x] Did you pull `master` into your branch?
- - [x] Did you *check for merge conflicts*?
- - [x] Did you *resolve* any that occurred? ***If you are having trouble or are confused, contact a lead!***
- [x] Did you fill out the below template?
- [x] Did you assign the right people for review (on the right)?

Arvind-Srinivasan commented on EAGLE Libraries/HyTechDevices.lbr:
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE eagle SYSTEM "eagle.dtd">
-<eagle version="9.6.2">
+<eagle version="9.5.0">
```
I think your eagle version is a little behind recent, as they went from 9.6.2 to 9.5.0.

Arvind-Srinivasan commented on EAGLE Libraries/HyTechDevices.lbr:
```diff
@@ -16604,6 +16769,82 @@ Demo Board</text>
 <text x="0" y="23.368" size="1.778" layer="95">&gt;NAME</text>
 <text x="0" y="-2.032" size="1.778" layer="95">&gt;VALUE</text>
 </symbol>
+<symbol name="RES3">
```
Could you use a more descriptive name for the symbol cause "RES3" doesn't explain it very well.
In addition, you seem to have lost the symbol Leo made for his resistor array.

Arvind-Srinivasan commented on EAGLE Libraries/HyTechDevices.lbr:
```diff
@@ -25662,6 +25903,101 @@ Dimensions: 5 mm x 20 mm
 </device>
 </devices>
 </deviceset>
+<deviceset name="YC248-JR-0710KL" prefix="R">
```
The device has multiple variants that all don't need to exist, look at Leo's CAT16 part I think for a decent single size for the array.

19etweinstock commented:
Made requested changes:

- Updated EAGLE to 9.6.2
- Deleted to RES3 part as it was unnecessary
- Set the YC248 to only have one package
- Leo's resistor array is still there

shaand1 commented:
Why not remove the device that this replaces? Or at least include in the description of Leo's CAT16 to use this new one?

19etweinstock commented:
Updated CAT16 part (the old resistor array) to reflect the change requested by Shaan
